testsuites/testsuites.lisp

```diff
@@ -1131,3 +1132,9 @@ just passed to the QUICKLISP:QUICKLOAD."
     (ql:quickload :cl+ssl.test)
     (run-fiveam-test-suite :cl+ssl)))
 
+(defmethod libtest ((library-name (eql :shasht)))
+  ;; The test framework used: parachute
+  (ql:quickload :shasht)  ; without this Quicklisp can't find :shasht/test
+  (ql:quickload :shasht/test)
+  (asdf:test-system :shasht))

```

Comment:
But `asdf:test-system` does not return any value, right?

Reply:
Okay, so I guess there has to be test adapter like `run-fiveam-test-suite`?

I am using parachute and there doesn't appear to be one already. I can work one into this PR if you want.

Reply:
I would gladly accept ad adapter and this test suite.